--- 2d/package/dist/index.full.js
+++ 8dec/package/dist/index.full.js
@@ -108081,7 +108081,7 @@
       easing: { type: String, required: false },
       disabled: { type: Boolean, required: false },
       appear: { type: Boolean, required: false, default: false },
-      mode: { required: false },
+      mode: { type: null, required: false },
       suppressAppearDuringHydration: { type: Boolean, required: false, default: false }
     },
     setup(__props) {
